test/usersController.test.js

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable no-underscore-dangle */
+const http = require('http');
+const chai = require('chai');
+const express = require('express');
+const chaiHttp = require('chai-http');
+const { OK, INTERNAL_SERVER_ERROR, NOT_FOUND } = require('http-status-codes');
+const expressRoutesRegistrar = require('express-routes-registrar');
+const db = require('../mongodb');
+const { usersRoutes } = require('../routes');
+const { usersController } = require('../controllers');
+const { Users } = require('../models');
+
+const { expect } = chai;
+
+chai.use(chaiHttp);
+
+describe('Users Routes', () => {
+  const PORT = 3000;
+  const URL = `http://localhost:${PORT}`;
+  const app = express();
+  app.use(express.json());
+  const server = http.createServer(app);
+  const registrar = expressRoutesRegistrar(app);
+
+  before((done) => {
+    db.connect()
```

Comment:
Maybe this can be done `var requester = chai.request(app).keepOpen()` but I'm not sure
https://www.chaijs.com/plugins/chai-http/